backend/frontend/templates/frontend/user/board/js_exec_action_card_moved.html

```diff
@@ -0,0 +1,5 @@
+{% include "frontend/user/board/partial_card.html" %}
+
+<script>
+  moveCardToCorrectColumn("{{card.id}}", "{{card.status}}");
+</script>
```

Comment:
There is a file called view_partial_action_card_moved. It's content is:

```
{% include "frontend/user/board/partial_card.html" %}

<script>
    moveCardToCorrectColumn("{{card.id}}","{{card.status}}")
</script>
```

It is exactly the same as this file.

Can you please make it DRY